mllib/src/main/scala/org/apache/spark/ml/classification/Classifier.scala

```diff
@@ -192,12 +192,12 @@ abstract class ClassificationModel[FeaturesType, M <: ClassificationModel[Featur
 
   /**
    * Predict label for the given features.
-   * This internal method is used to implement `transform()` and output [[predictionCol]].
+   * This method is used to implement `transform()` and output [[predictionCol]].
    *
    * This default implementation for classification predicts the index of the maximum value
    * from `predictRaw()`.
    */
-  override protected def predict(features: FeaturesType): Double = {
+  override def predict(features: FeaturesType): Double = {
```

Comment:
Is @Since annotation needed here? @WeichenXu123

Reply:
I think no. It will inherit `@Since` from super class `predict` method.